client/components/recycleAndReuseComponents/AddItemDialog.js

```diff
@@ -0,0 +1,46 @@
+import { React, useRef } from 'react'
+import {
+    AlertDialog,
+    AlertDialogOverlay,
+    AlertDialogContent,
+    AlertDialogHeader,
+    AlertDialogBody,
+    AlertDialogFooter,
+    Button
+} from '@chakra-ui/react'
+
+const AddItemDialog = ({ setTabIndex, didSelectItems, willTriggerDialog }) => {
+    const cancelRef = useRef();
+    const onClose = () => willTriggerDialog(false);
+    const navigateToVerifyItem = () => {
+        setTabIndex(1);
+        onClose();
+    }
+
+    return (
+        <AlertDialog isOpen={didSelectItems} leastDestructiveRef={cancelRef} onClose={onClose}>
+            <AlertDialogOverlay>
+                <AlertDialogContent>
+                    <AlertDialogHeader fontSize='lg' fontWeight='bold'>
+                        Confirmation
+                    </AlertDialogHeader>
+
+                    <AlertDialogBody>
+                        Is every item listed? You cannot return to this page to add more items for this round!
+                    </AlertDialogBody>
+
+                    <AlertDialogFooter>
+                    <Button ref={cancelRef} onClick={onClose}>
+                        Cancel
+                    </Button>
+                    <Button colorScheme='green' onClick={navigateToVerifyItem} ml={3}>
+                        Confirm
+                    </Button>
+                    </AlertDialogFooter>
+                </AlertDialogContent>
+            </AlertDialogOverlay>
+        </AlertDialog>
+    )
+}
+
+export default AddItemDialog
```

Comment:
@theodorayy This is what you have written. I just refactored it into a separate component because this component is technically only used by `AddItem` and should not be part of the main tabs component. The code would be more readable with such refactoring (I think 😅 )

Let me know if you have any concerns/ questions

Reply:
Yeah that's good. Thanks!